### DISCUSSION TITLE
[Vite] Server log

### DISCUSSION THREAD

**Discussion by saschatimme** [remix-run/remix#7850] [Q&A] [answered]
With the previous remix dev server all incoming requests got logged to the terminal. Playing around with the new Vite support this is no more the case using the suggested start vite config. How can I enable this again?

**Comment by kiliman** [comment, marked as answer]
```bash
npm i -D morgan @types/morgan
```

```ts
// vite.config.ts

import { unstable_vitePlugin as remix } from "@remix-run/dev";
import morgan from "morgan";
import { defineConfig, type ViteDevServer } from "vite";
import tsconfigPaths from "vite-tsconfig-paths";

export default defineConfig({
  plugins: [morganPlugin(), remix(), tsconfigPaths()],
});

function morganPlugin() {
  return {
    name: "morgan-plugin",
    configureServer(server: ViteDevServer) {
      return () => {
        server.middlewares.use(morgan("tiny"));
      };
    },
  };
}
```
<img width="520" alt="image" src="https://github.com/remix-run/remix/assets/47168/95547d22-b2bb-4787-9cee-6b3bf2cde6b4">

**Reply by saschatimme** [reply]
Thank you 👍 . Wouldn't this make sense to be included by default or at least mentioned in the docs?

**Reply by kiliman** [reply]
Vite support is new. They're still learning the process as well. And they are listening to feedback.

**Reply by sergiodxa** [reply]
I think if it's that simple to add the logger it could come on a new app but not included in `remix` plugin itself, so you can customize it.

**Reply by lifeiscontent** [reply]
@kiliman it doesn't seem that the data requests get logged, e.g. all loaders/actions that were run in remix without vite would log, but in this scenario the loaders that are run are not logging

in my scenario I have a loader that creates an svg and the console.log of that loader is not being called with rendering

```<img src="/resources/example.svg" />```

where this `/resources/example[.svg].tsx` exports a `loader` function any ideas on how to get the equivalent logging back?
